pom.xml

```diff
@@ -40,6 +39,30 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
+			<groupId>io.springfox</groupId>
+			<artifactId>springfox-swagger-ui</artifactId>
+			<version>${api.doc.swagger}</version>
+		</dependency>
+		<dependency>
+			<groupId>io.springfox</groupId>
+			<artifactId>springfox-swagger2</artifactId>
+			<version>${api.doc.swagger}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.projectlombok</groupId>
+			<artifactId>lombok</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-devtools</artifactId>
+			<optional>true</optional>
+		</dependency>
+		<dependency>
+			<groupId>junit</groupId>
```

Comment:
nadal nie dogadaliśmy się z tym junitem.
Wyzej w pomie masz:
```
			<exclusions>
				<exclusion>					
					<groupId>org.junit.vintage</groupId>	
					<artifactId>junit-vintage-engine</artifactId>	
				</exclusion>
			</exclusions>	
```
oznacza to że chcesz używac tylko i wyłącznie JUnit 5. 
A pare linijek niżej dodajesz JUnit 4 jako dependency - coś tu jest nie tak.
Tak jak mówiłem stary Junit nie jest Ci potrzebny